lib/dice/src/alias_cert_tmpl.rs

```diff
@@ -0,0 +1,111 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// NOTE: This DER blob, offsets & lengths is mostly generated code. This was
+// accomplished by creating a certificate with the desired structure (including
+// DICE specific extensions and policy) using the openssl ca tools. The fields
+// that we need to operate on were identified and their offsets recorded. The
+// values in these regions (signatureValue, serialNumber, issuer, subject,
+// validity etc) are then removed.
+//
+// TODO: generate cert template DER from ASN.1 & text config
+#[allow(dead_code)]
+pub const SIZE: usize = 608;
+#[allow(dead_code)]
+pub const SERIAL_NUMBER_START: usize = 15;
```

Comment:
Nit: these could be more compactly expressed as ranges, e.g.
```rust
pub const SERIAL_NUMBER_RANGE: core::ops::Range<usize> = 15..16;
```

Doing so would reduce the potential for copy-paste errors!

Reply:
TIL `core::ops::Range` exists. The code that generates these tmpl files can't go into the hubris tree cleanly since I'm using a bunch of external tools to do ASN.1 / x509 crimes but I'll queue this change up for the code gen. This may also help w/ your previous point about `_LENGTH` values.

Reply:
https://github.com/oxidecomputer/hubris/issues/747